front/src/pods/trainer/components/session.component.tsx

```diff
@@ -30,8 +34,14 @@ const getFullContent = (currenSessionContent: string) => {
 export const SessionComponent: React.FC<Props> = props => {
   const { log, handleSendFullContentLog, className } = props;
 
+  const [autoScroll, setAutoScroll] = React.useState(false);
```

Comment:
Same changes as in previous file.
And once you are done we will extract this to a generic hook 😇